ViewModel/Study/PropertiesVM.cs

```diff
@@ -25,6 +22,9 @@ public class PropertiesVM : Editors.BaseEditorVM
         private Single _UpdatedPriceIndex;
         #endregion
         #region Properties
+
+        public double DiscountRate { get; set; }
+        public int PeriodOfAnalysis { get; set; }
```

Comment:
These are probably going to be editable, so I'd recommend updating the setter to use NotifyPropertyChanged.

Reply:
done.